tenable/io/v3/base/endpoints/explore.py

```diff
@@ -148,6 +153,150 @@ def _search(self,
             _payload=payload
         )
 
+    def _search_was(self,
+                    *,
+                    resource: str,
+                    api_path: str,
+                    api_version: int = 1,
+                    sort_type: Enum = _sort_type.default,
+                    return_resp: bool = False,
+                    iterator_cls: ExploreIteratorWas = SearchIteratorWas,
```

Comment:
Let's rename SearchIteratorWas -> SearchIteratorWAS
and SearchWasSchema  -> SearchWASSchema